docs/cpp/customize-cpp-settings.md

```diff
@@ -0,0 +1,174 @@
+---
+ContentId: 4E34F6AF-BFC6-4BBB-8464-2E50C85AE826
+DateApproved: 4/25/2025
+MetaDescription: How to customize the c_cpp_properties.json file for the C++ extension.
+---
+# C++ extension settings reference
+
+The C++ extension settings are highly configurable. This article explains the schema for the `c_cpp_properties.json` file. For general information about settings in VS Code, refer to [Configure settings](/docs/getstarted/personalize-vscode.md#configure-settings), as well as the [Variables reference](/docs/reference/variables-reference.md) and [Default VS Code Settings](/docs/reference/default-settings.md).
```

Comment:
Is the `Default VS Code Settings` supposed to match the title in the doc `Default settings reference`?